app/src/test/java/com/example/parkingsystem/mvp/presenter/ReservationPresenterTest.java

```diff
@@ -0,0 +1,135 @@
+package com.example.parkingsystem.mvp.presenter;
+
+import android.app.DatePickerDialog;
+import android.app.TimePickerDialog;
+import com.example.parkingsystem.mvp.contract.ParkingContract;
+import com.example.parkingsystem.mvp.contract.ReservationContract;
+import com.example.parkingsystem.mvp.model.ReservationModel;
+import com.example.parkingsystem.mvp.model.reservation.Reservation;
+import com.example.parkingsystem.mvp.model.reservation.ReservationInformationDB;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.List;
+import static org.junit.Assert.assertEquals;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import org.mockito.internal.matchers.Any;
+import org.mockito.junit.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.StrictStubs.class)
+public class ReservationPresenterTest {
+
+    public static final String SECURITY_CODE = "1234";
+    public static final String PLACE = "10";
+    public static final String EMPTY_STRING = "";
+
+    public static final int YEAR_START = 2021;
+    public static final int MONTH_START = 7;
+    public static final int DAY_OF_MONTH_START = 8;
+    public static final int HOUR_START = 3;
+    public static final int MINUTE_START = 17;
+
+    public static final int YEAR_FINISH = 2022;
+    public static final int MONTH_FINISH = 9;
+    public static final int DAY_OF_MONTH_FINISH = 10;
+    public static final int HOUR_FINISH = 6;
+    public static final int MINUTE_FINISH = 25;
+
+    @Mock
```

Comment:
please use only one type of mock.
Use the annotation `@Mock` to the  `= mock(class)`

Reply:
fixed